corehq/apps/reports/standard/cases/basic.py

```diff
@@ -28,7 +29,7 @@
 from .data_sources import CaseDisplayES
 
 
-class CaseListMixin(ElasticProjectInspectionReport, ProjectReportParametersMixin):
+class CaseListMixin(ESQueryProfilerMixin, ElasticProjectInspectionReport, ProjectReportParametersMixin):
```

Comment:
Oops, need to be removed here. It's not applicable for this PR yet. Only did it for testing locally.